boot/boot_serial/src/boot_serial.c

```diff
@@ -1467,7 +1478,7 @@ static int boot_serial_get_hash(const struct image_header *hdr,
             break;
         }
 
-        if (type == IMAGE_TLV_SHA256) {
+        if (type == IMAGE_SHA_TLV) {
             /* Get the image's hash value from the manifest section. */
             if (len != 32) {
```

Comment:
length not updated

Reply:
Funny how it worked with this bugs, as I was able to get image list and mark image for test.